src/components/common/landingPage/trending/trending.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getTrendingArticles } from '../../../../redux/actions/landingPageActions';
+import './trending.scss';
+
+const formatDate = (createdAt) => {
+  const date = new Date(createdAt);
+  return `${date.toLocaleString('default', { month: 'short' })} ${date.getDay()}`;
+};
+
+const TrendingArticles = (props) => {
+  // eslint-disable-next-line react/prop-types
```

Comment:
The way you _conveniently_ disables rules shaa. Why is this one here?

Reply:
I'm sorry, I would fix all the disabled rules. I was just trying to make the code work first.